### DISCUSSION TITLE
Giving me the username in a black text which is nearly impossible wondering why it is doing so and wanna change it

### DISCUSSION THREAD

**Discussion by sridhar-5** [starship/starship#4901] [Q&A]
<img width="675" alt="Screenshot 2023-02-13 at 4 10 43 PM" src="https://user-images.githubusercontent.com/65086710/218436445-ac311de7-4646-4428-8c03-d608eba3bcec.png">

In this image after user: the username is in white bold as expected in the format but I was wondering where is the `with username` after the username coming from I don't have it in my starship.toml file. can someone help me figure this out ?

this is my username config in starship.toml

<img width="675" alt="Screenshot 2023-02-13 at 4 13 06 PM" src="https://user-images.githubusercontent.com/65086710/218436911-34a6d5b6-3d7e-405e-bc34-88078da554a8.png">

**Comment by sk9073** [comment]
Yeah i faced the same issue ! , abandoned the package at the end! please resolve this issue!!!

**Comment by rohithmsr** [comment]
I am facing the same thing right now thought may be changing the color of normal text would work because the text after the username meaning `with $username` that is being rendered isn't in starship.toml file i checked it twice. Didn't make sense to me.

**Reply by sridhar-5** [reply]
Exactly

**Comment by davidkna** [comment]
Starship is using the corresponding ANSI color code to display `white` which depends on your terminal color scheme. Please ensure that it maps the color to a white color.
Furthermore, try using the related `bright-white` color instead. Or, to set the foreground color independently of the terminal scheme, use a hex color like `#ffffff`.

**Comment by mickimnet** [comment]
@sridhar-5, regarding your question, where the `with $username` is coming from, could you provide your `format` section of your configuration, please?
